tinygrad/tensor.py

```diff
@@ -199,7 +198,7 @@ def backward(self):
 
   def reshape(self, shape, *args) -> Tensor:
     new_shape = argfix(shape, *args)
-    assert len(new_shape) > 0 and all(x != 0 for x in new_shape), f"zeros not allowed in shape {new_shape}"
+    assert len(new_shape) == 0 or all(x != 0 for x in new_shape), f"zeros not allowed in shape {new_shape}"
```

Comment:
I don't think this first clause in the or is required.

Reply:
Got it.